src/main/java/core/basesyntax/Pen.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Pen {
+    String template = "Figure: %s, area: %.1f sq.units, %s, color: %s";
```

Comment:
```suggestion
    String TEMPLATE = "Figure: %s, area: %.1f sq.units, %s, color: %s";
```

Reply:
I think that we must use NAME_STYlE only for number constant

Reply:
Why?)

Reply:
*I thoutht.